SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -320,25 +340,50 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 		return;
 	}
 
-	FClassInfo* Info = TypebindingManager->FindClassInfoByClass(Params->TargetObject->GetClass());
-	if (Info == nullptr)
+	FClassInfo Info;
+
+	// If Subobject
+	if (AActor* Actor = Cast<AActor>(Params->TargetObject->GetOuter()))
```

Comment:
Can we change this to first case being `Params->TargetObject->IsA<AActor>()`, otherwise doing
`check(Params->TargetObject->GetOuter()->IsA<AActor>())`?